VideoIndexerEnabledByArc/aks/vi_extension_install.sh

```diff
@@ -230,13 +231,13 @@ if [[ $install_aks_cluster == "true" ]]; then
   fi
 
   echo -e "\t create aks cluster Name: $aks , Resource Group $rg- ***done***"
-  echo "Adding another workload node pool"
+  echo "Adding two new node pool types, workload and workloadf32"
```

Comment:
I am not sure this sample needs to be changed by adding new node pool
this is a very dedicated scenario for summarization 
will all customers have summarization by default ? 

cant we use the Existing node pools and make them be F32 instead of adding another node pool ?